pkgs/applications/office/pympress/default.nix

```diff
@@ -6,19 +6,37 @@
 , gobject-introspection
 , libcanberra-gtk3
 , poppler_gi
+, pango
+, harfbuzz
+, gdk-pixbuf
+, atk
 , withGstreamer ? stdenv.isLinux
 , withVLC ? stdenv.isLinux
  }:
 
 python3Packages.buildPythonApplication rec {
   pname = "pympress";
-  version = "1.7.0";
+  version = "1.7.2";
 
   src = python3Packages.fetchPypi {
     inherit pname version;
-    sha256 = "sha256-AxH0PyAWYEEIqQAx9gG2eYyXMijLZGZqXkRhld32ieE=";
+    sha256 = "LFUzrGHr8jmUqoIcKokC0gNDVmW1EUZlj9eI+GDycvI=";
   };
 
+  makeWrapperArgs = [
+    "--set GI_TYPELIB_PATH ${lib.makeSearchPathOutput "lib"
+      "lib/girepository-1.0" ([
+        gtk3
+        pango
+        harfbuzz
+        gdk-pixbuf
+        atk
+      ])}:${lib.makeSearchPathOutput "out"
+      "lib/girepository-1.0" ([
+        poppler_gi
+      ])}"
+  ];
```

Comment:
This should all be done automatically if gobject-introspection is moved to nativeBuildInputs

Reply:
Good catch, thank you! Pol is away right now, so I've fixed that. It appears to work fine with your suggested change

Reply:
You are sure we do not need to add the other packages to buildInputs? If yes then please remove them from inputs again.

Reply:
> You are sure we do not need to add the other packages to buildInputs? If yes then please remove them from inputs again.

It seems to be working fine on my machine without adding the other packages to buildInputs, although this does seem to contradict with some of the errors we were getting earlier while making up the `makeWrapperArgs`. Running `nix why-depends` does seem to show them as dependencies though, so I suspect that this is why gobject-introspection can still pick them up

(e.g. for atk)
```
/nix/store/976jflnzg5ias7l0r91z4m70w645d6lg-pympress-1.7.2
├───/nix/store/gnqjdvvy484fnhrj4cfwkq9mamkgf6mp-atk-2.38.0
└───/nix/store/2bs64xwzj41afvk5h7hzma6dp0sx83sx-gtk+3-3.24.34
    ├───/nix/store/gnqjdvvy484fnhrj4cfwkq9mamkgf6mp-atk-2.38.0
    └───/nix/store/6vz3f3wzvcbq1p9z6lvl0znispi7xz00-at-spi2-atk-2.38.0
        └───/nix/store/gnqjdvvy484fnhrj4cfwkq9mamkgf6mp-atk-2.38.0```